ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/siteConfig/siteConfigBuilder.go

```diff
@@ -53,6 +53,14 @@ func (scbuilder *SiteConfigBuilder) Build(siteConfigTemp SiteConfig) (map[string
 		if cluster.ClusterName == "" {
 			return clustersCRs, errors.New("Error: Missing cluster name at site " + siteConfigTemp.Metadata.Name)
 		}
+		if cluster.NetworkType == "" {
```

Comment:
If the user does not specify a network type in the SiteConfig YAML what will be populated into the annotation?

Reply:
OVNKubernetes